Added instructions on current progress and total number of items.

windwhim opened the pull request:
Added instructions on current progress and total number of items.

windwhim committed:
Update app.py

Added instructions on current progress and total number of items.

windwhim committed:
Update app.py

Fixed a bug where the total number of items was 1 when opened a folder.

maplelost commented:
Wonderful idea! But you don't need to add an extra **self.filenames** to record the inde of the file you can just use the pyqt api.
Like:
```python
    def setClean(self):
        self.dirty = False
        self.actions.save.setEnabled(False)
        self.actions.createMode.setEnabled(True)
        self.actions.createRectangleMode.setEnabled(True)
        self.actions.createCircleMode.setEnabled(True)
        self.actions.createLineMode.setEnabled(True)
        self.actions.createPointMode.setEnabled(True)
        self.actions.createLineStripMode.setEnabled(True)
        self.actions.createAiPolygonMode.setEnabled(True)
        self.actions.createAiMaskMode.setEnabled(True)
        title = __appname__
        if self.filename is not None:
            # extra 添加文件进度
            file_index = self.fileListWidget.currentRow()
            title = f"{title} - {self.filename} ({file_index + 1}/{self.fileListWidget.count()})"
            # extra end
        self.setWindowTitle(title)

        if self.hasLabelFile():
            self.actions.deleteFile.setEnabled(True)
        else:
            self.actions.deleteFile.setEnabled(False)
```

windwhim commented:
这样代码更加优雅了

windwhim committed:
Update app.py

Use a better way to display progress, but when opening an image, it will display 0/0.

windwhim committed:
Merge branch 'wkentaro:main' into main

windwhim committed:
Merge branch 'wkentaro:main' into main

wkentaro commented:
> *This was generated by AI during triage.*

Thanks for this contribution! The file progress indicator (current/total) you added is a good idea and has since been implemented in `main` via the `_get_window_title()` refactor (see `labelme/app.py` line 1223). Your PR now has merge conflicts and the functionality is already covered, so I'm closing this as superseded. Thanks again for the patch!

wkentaro commented:
> *This was generated by AI during PR processing.*

Thanks for this. The current window title already shows the image progress (`[index+1/count]`) via a dedicated `_format_window_title` helper that also handles edge cases, so this is effectively already implemented and the branch now conflicts. Recommending close (your call to review or close).
